runtime/planner.js

```diff
@@ -221,7 +221,8 @@ Planner.ResolutionStrategies = [
   SearchTokensToHandles,
   GroupHandleConnections,
   FallbackFate,
-  CreateHandles,
+  // #1671: requiring 'create' fate to be explicit.
+  // CreateHandles,
```

Comment:
Maybe just delete this line

Reply:
then maybe it's better to delete CreateHandles strategy altogether?